rmw_fastrtps_cpp/Doxyfile

```diff
@@ -0,0 +1,35 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rmw_fastrtps_cpp"
+PROJECT_NUMBER         = master
+PROJECT_BRIEF          = "Implement the ROS middleware interface using eProsima FastRTPS static code generation in C++."
+
+# Use these lines to include the generated logging.hpp (update install path if needed)
```

Comment:
This looks like a copy-paste error.

Reply:
https://github.com/ros2/rmw_fastrtps/pull/372/commits/1f7761e2cf2a0f9117bf0c81b6ad9ea80cb1c0b8